compiler/damlc/tests/daml-test-files/Interface.daml

```diff
@@ -9,6 +9,7 @@ import DA.Assert ((===))
 
 -- | An interface comment.
 interface Token where
+  viewtype ()
```

Comment:
this made me wonder, would users get an error for trying to define a method named `view`?